### PR TITLE
Adding New Notebook 

### PR DESCRIPTION
**Pull request by lv6520**
*Issue #, if available:*

*Description of changes:*
This is adding a new notebook which is  a Sample Inventory Planning which is using the RETAIL Domain 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by chrisking** [comment]
@akhilrazdan looks ready to ship, please merge.

File: notebooks/BikeShareExample.ipynb [hunk 1]
```diff
@@ -622,14 +622,27 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Generate Plot \n",
+    "print(\"Forcast for P90\") \n",
+    "d = pd.DataFrame.from_dict(forecastResponse['Forecast'])\n",
+    "df = pd.DataFrame.from_dict(d.loc['mean']['Predictions']).dropna().rename(columns = {'Val':'p90'})\n",
```
Comment (IronistM) [inline notebooks/BikeShareExample.ipynb]: @lv6520  You take the `mean` column but you label it as `P90`. It's a typo right?

File: notebooks/BikeShareExample.ipynb [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Generate Plot \n",
+    "print(\"Forcast for P90\") \n",
```
Comment (IronistM) [inline notebooks/BikeShareExample.ipynb]: Should be `Forecast for mean prediction`?

File: notebooks/BikeShareExample.ipynb (hunk 1 above)
Comment (lv6520) [reply notebooks/BikeShareExample.ipynb]: yes .. fixed

File: notebooks/BikeShareExample.ipynb (hunk 2 above)
Comment (lv6520) [reply notebooks/BikeShareExample.ipynb]: I have fixed the print statement

**Pull request merged**